### ISSUE TITLE
There are 1 parameter(s) but only 0 format argument(s) were provided.

### ISSUE THREAD

**Issue by Sammers21** [artipie/artipie#311]
Artipie startup commad
```bash
$  docker run -it -v $(pwd)/artipie.yaml:/etc/artipie.yml -v $(pwd):/var/artipie  -p 8080:80 artipie/artipie:0.8.9
```

Dir structure:
```bash
$ tree .
.
├── artipie.yaml
├── configs
│   └── repo.yaml
├── data
└── repo
    └── white-paper.pdf
```

__`artipie.yaml`__:
```yaml
meta:
  storage:
    type: fs
    path: /var/artipie/configs
  metrics:
    type: log
  permissions:
    "*":
      - "*"
```

__`repo.yaml`__:
```yaml
repo:
  type: file
  storage:
    type: fs
    path: /var/artipie/data
  permissions:
    "*":
      - "*"
```

Bug occurence:
```bash
$ curl  -i -X GET  http://localhost:8080/repo/white-paper.pdf
HTTP/1.1 500 Internal Server Error
content-length: 4922

java.lang.IllegalArgumentException: There are 1 parameter(s) but only 0 format argument(s) were provided.
java.lang.IllegalArgumentException: There are 1 parameter(s) but only 0 format argument(s) were provided.
	at com.jcabi.log.PreFormatter.process(PreFormatter.java:148)
	at com.jcabi.log.PreFormatter.<init>(PreFormatter.java:83)
	at com.jcabi.log.Logger.format(Logger.java:123)
	at com.jcabi.log.Logger.error(Logger.java:247)
	at com.artipie.http.SafeSlice$RsSafe.send(SafeSlice.java:100)
	at com.artipie.ResponseMetricsSlice.lambda$response$1(ResponseMetricsSlice.java:80)
	at com.artipie.vertx.VertxSliceServer.serve(VertxSliceServer.java:169)
	at com.artipie.vertx.VertxSliceServer.lambda$proxyHandler$1(VertxSliceServer.java:145)
	at io.vertx.reactivex.core.http.HttpServer$1.handle(HttpServer.java:116)
	at io.vertx.reactivex.core.http.HttpServer$1.handle(HttpServer.java:114)
	at io.vertx.core.http.impl.WebSocketRequestHandler.handle(WebSocketRequestHandler.java:50)
	at io.vertx.core.http.impl.WebSocketRequestHandler.handle(WebSocketRequestHandler.java:32)
	at io.vertx.core.http.impl.Http1xServerConnection.handleMessage(Http1xServerConnection.java:136)
	at io.vertx.core.impl.ContextImpl.executeTask(ContextImpl.java:369)
	at io.vertx.core.impl.EventLoopContext.execute(EventLoopContext.java:43)
	at io.vertx.core.impl.ContextImpl.executeFromIO(ContextImpl.java:232)
	at io.vertx.core.net.impl.VertxHandler.channelRead(VertxHandler.java:173)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at io.netty.handler.codec.http.websocketx.extensions.WebSocketServerExtensionHandler.channelRead(WebSocketServerExtensionHandler.java:101)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.vertx.core.http.impl.Http1xUpgradeToH2CHandler.channelRead(Http1xUpgradeToH2CHandler.java:109)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:323)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:297)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.vertx.core.http.impl.Http1xOrH2CHandler.end(Http1xOrH2CHandler.java:61)
	at io.vertx.core.http.impl.Http1xOrH2CHandler.channelRead(Http1xOrH2CHandler.java:38)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:357)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:379)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:365)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:163)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:714)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:576)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:493)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:989)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Unknown Source)
```

**Comment by g4s8** [comment]
@rultor release, tag=`0.8.11`

**Comment by g4s8** [comment]
@Sammers21 should be fixed, try `0.8.11`
